tests/behat/behat-features/src/MediaReplicationContext.php

```diff
@@ -0,0 +1,168 @@
+        if ($found_element) {
+          throw new ExpectationException('Element should not be found.', $this->getSession());
+        }
+        break;
+      }
+    }
+  }
+
+  /**
+   * @Given I click on :index of created media in :entity_browser
+   */
+  public function iClickOnIndexOfCreatedMediaInEntityBrowser($index, $entity_browser) {
+    $id = array_slice(array_keys(self::$createdMediaEntities), $index, 1)[0];
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").closest(\".views-row\").click()");
+  }
+
+  /**
+   * @Given I click on edited media in :entity_browser
+   */
+  public function iClickOnCurrentOfCreatedMediaInEntityBrowser($entity_browser) {
+    $id = self::$currentlySelectedMedia;
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$id\").closest(\".views-row\").click()");
+  }
+
+  /**
+   * @Given I click on unused media in :entity_browser
+   */
+  public function iClickOnUnsedMediaInEntityBrowser($entity_browser) {
+    $unused = self::$unusedMedia;
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$unused\").length > 0");
+    if (!$found_element) {
+      throw new ExpectationException('Element not found.', $this->getSession());
+    }
+    $this->getSession()
+      ->evaluateScript("jQuery(\"#entity_browser_iframe_$entity_browser\").contents().find(\"#edit-entity-browser-select-media$unused\").closest(\".views-row\").click()");
+  }
+
+  /**
+   * I edit last selected media.
+   *
+   * @Then I edit last media
+   */
+  public function editLastMedia() {
+    $id = self::$currentlySelectedMedia;
+    $xpath = "//td[contains(@class, 'field-operations')]//a[contains(@href, 'media/$id/edit')]";
+    $this->getSession()->getDriver()
+      ->click($xpath);
+  }
+
+  /**
+   * @Then I should see title of currently selected media
+   */
+  public function currentlySelectedVisible() {
+    $title = self::$createdMediaEntities[self::$currentlySelectedMedia];
+    $found_element = $this->getSession()
+      ->evaluateScript("jQuery(\"*:contains('$title')\").length > 0");
```

Comment:
that jQuery hacks should only be used when really necessary. Did the mink element API not work here?
If some mink step definition is missing, that is perfect to add to our behat-drupal-utils